manifests/init.pp

```diff
@@ -241,8 +241,8 @@
 ) {
 
   # Fail on unsupported platforms
-  if $facts['os']['family'] == 'RedHat' and !($facts['os']['release']['major'] in ['5','6','7','8']) {
-    fail("osfamily RedHat's os.release.major is <${::facts['os']['release']['major']}> and must be 5, 6, 7 or 8")
+  if $facts['os']['name'] =~ 'RedHat' and !($facts['os']['release']['major'] in ['2','5','6','7','8']) {
```

Comment:
Why the change from `==` to `=~` ? I don't think that is necessary.

Reply:
That is a mistake. I was going to try and make a regex to resolve the os name but I thought that might create new problems in other areas so I left it alone. It can be changed back to == with no issues.